common/src/main/java/com/revenuecat/purchases/common/verification/SigningManager.kt

```diff
@@ -51,10 +59,24 @@ class SigningManager(
         val signatureToVerify = decodedSignature.copyOfRange(SALT_BYTES_SIZE, decodedSignature.size)
         val messageToVerify = saltBytes + decodedNonce + requestTime.toByteArray() + signatureMessage.toByteArray()
         val verificationResult = signatureVerifier.verify(signatureToVerify, messageToVerify)
+
         return if (verificationResult) {
             HTTPResult.VerificationStatus.SUCCESS
         } else {
+            errorLog(NetworkStrings.VERIFICATION_ERROR)
             HTTPResult.VerificationStatus.ERROR
         }
     }
+
+    private fun getVerifier(): SignatureVerifier? {
```

Comment:
Maybe this should be defined in `SignatureVerificationMode`? (or an extension of that type).